### ISSUE TITLE
[test] cargo test with `--show-output` report error

### ISSUE THREAD

**Issue by ariesdevil** [datafuselabs/datafuse#1096]
**Summary**
When we run cargo test with argument `--show-output`, get the error following:
```
error: Found argument '--show-output' which wasn't expected, or isn't valid in this context

USAGE:
    fuse_query-037e5cb8b0db2c93 [OPTIONS]

For more information try --help
```

More about usage of show output pls see: https://doc.rust-lang.org/book/ch11-02-running-tests.html#showing-function-output

**Comment by drmingdrmer** [comment]
This may be the same issue as:
https://github.com/datafuselabs/datafuse/pull/897

What did you do when you get this error?

**Comment by ariesdevil** [comment]
@drmingdrmer Just run `cargo test -- --show-output` at project root dir

**Comment by drmingdrmer** [comment]
fuse-store does not have this problem: 
`cargo test -p fuse-store -- --show-output` passed happily.

@sundy-li @zhang2014 
It should be a problem with fuse-query config parser. 
Maybe it is the fault of `StructOptToml`, as I mentioned in fuse store config:
```rust
impl Config {
    /// StructOptToml provides a default Default impl that loads config from cli args,
    /// which conflicts with unit test if case-filter arguments passed, e.g.:
    /// `cargo test my_unit_test_fn`
    ///
    /// Thus we need another method to generate an empty default instance.
    pub fn empty() -> Self {
        Self::from_iter(&Vec::<&'static str>::new())
    }
}
```

**Comment by sundy-li** [comment]
Should disable `from_args` and `from_args_with_toml` in cargo test, but I did not find a way to make it.

**Comment by PsiACE** [comment]
Solved in #1110, by ignoring some tests
